docs/getting_started/using_modin/using_modin_cluster/using_modin_ray_cluster.rst

```diff
@@ -0,0 +1,132 @@
+Using Modin on Ray in a Cluster
+===============================
+
+.. note::
+  | *Estimated Reading Time: 15 minutes*
+
+Often in practice we have a need to exceed the capabilities of a single machine.
+Modin works and performs well in both local mode and in a cluster environment.
+The key advantage of Modin is that your python code does not change between
+local development and cluster execution. Users are not required to think about
+how many workers exist or how to distribute and partition their data;
+Modin handles all of this seamlessly and transparently.
+
+.. note::
+   You can also use a Jupyter notebook, but you need to deploy a Jupyter server 
+   on the remote cluster head node and connect to it.
+
+.. image:: ../../../img/modin_cluster.png
+   :alt: Modin cluster
+   :align: center
+
+Extra requirements for AWS authentication
+-----------------------------------------
+
+First of all, install the necessary dependencies in your environment:
+
+.. code-block:: bash
+
+   pip install boto3
+
+The next step is to setup your AWS credentials. One can set  ``AWS_ACCESS_KEY_ID``, 
+``AWS_SECRET_ACCESS_KEY`` and ``AWS_SESSION_TOKEN`` environment variables or  
```

Comment:
Can we add a link to some page that explains the appropriate setting of these env vars?

Reply:
done